### ISSUE TITLE
[LSF] Idées à communiquer à l'équipe en charge du livret handicap

### ISSUE THREAD

**Issue by LVAI** [jlm2017/jlm-video-subtitles#146]
Bonjour à tous,

Je fais une autre issue pour permettre à tous d'exprimer toutes les idées qui pourraient améliorer la vie des Sourds et en débattre.

En parallèle il y a le sondage qui récolte les avis extérieurs, voir issue #145 

Ces deux sources nous permettront de rédiger un document à destination de l'équipe en charge de rédiger le livret sur le handicap. Selon moi, le livret devra comporter :
1. Une introduction sur les spécificités de la communauté Sourde (ex : différence entre sourds et malentendants, le fait qu'ils soient autonomes et ne ressentent aucun handicap, leur combat pour être directement acteurs de ce qui les concerne et contre la régression de la LSF, des ouvrages de référence, etc.)
2. L'explication de notre démarche, notre équipe, comment on a procédé, graphique du sondage
3. L'itération des différentes idées à plus ou moins long terme, détaillées et regroupées par domaines
4. Une annexe avec les résultats du sondage.

Voilà.
A vos idées !!

**Comment by NoelleM** [comment]
Brillante initiative ! Bravo :)
Lorie, attendons-nous le dépouillement du questionnaire et la liste des "revendications" (légitimes) des principaux concernés avant de mettre nos propres propositions (pour éviter les doublons ou les erreurs de besoins) ?

**Comment by LVAI** [comment]
Bon, du coup je m'y colle :)

Il y aura peut-être des redites avec les tchat Mattermost et Facebook mais le but c'est de tout concentrer ici.

Ensuite, on est tous d'accord que les Sourds veulent (et c'est bien normal) s'exprimer eux-mêmes et parler pour eux. Et qu'on ne veut pas refaire un congrès de Milan*. Mais ça ne nous empêche de proposer des idées. Le but, et on le mettra en avant, c'est de les faire valider à terme par les Sourds eux-mêmes. Beaucoup de ceux qui ont répondu au sondage souhaitent en connaître le résultat. Ce sera l'occasion de leur communiquer notre document et, pourquoi pas, un nouveau sondage pour recueillir leur avis final (un peu comme la constituante, ils seraient l'alpha et l'oméga !).

Je pense que la première chose à faire (court terme) c'est de permettre au Sourds d'avoir accès à de l'information :
1. Encart et sous-titres sur les JT des chaines publiques
2. Remboursement des interprètes pour les démarches
3. Informations visuelles dans les lieux publiques
(nous irons aux bois...)

Ensuite à moyen terme, continuer dans ce sens et promulguer la Langue des signes :
4. Recrutement d’interprètes fonctionnaires pour servir dans les lieux publics et les administrations
5. Promotion des RDV vidéos avec interprète (j'ai vu ça dans une banque)
6. Possibilité d'apprendre la LSF au collège et lycée (LV2)

Et à plus long terme, concrétisation de l'intégration des Sourds français :
7. Reconnaissance de la Langue des Signes comme seconde langue officielle du pays
8. Enseignement de la LSF à l'école (français lu, écrit, parlé, signé)
9. Présence systématique d'un traducteur dans toutes les émissions télévisuelles

Vaste programme ^^ 

*Pour ceux qui ne savent pas, le congrès de Milan est un point noir de l'histoire Sourde. Troisième congrès international pour l'amélioration du sort des sourds-muets, il a acté l'adoption en Europe des méthodes d'enseignement exclusivement orale et l'interdiction de la langue des signes. Le congrès rassemble 250 entendants de toute l'Europe, 3 Sourds, aucun interprète.

**Comment by NoelleM** [comment]
"Remboursement des interprètes pour les démarches" : attention, les personnes sourdes touchent déjà une allocation, versée d'office (et sans contrôle de son utilisation) pour ce genre de prestations...Mais la somme forfaitaire est nettement insuffisante et ne couvre qu'une seule ou deux interventions d'interprètes professionnels (dont les prestations sont fort chères) !

J'opterais plutôt pour la création d'une vraie formation à 2 niveaux :
- interface de communication du service public : pour l'accompagnement dans les formalités etc. et la traduction en LSF
- interprète en LSF, également agent du service public :  pour les (nouvelles) missions régaliennes (santé, justice, éducation, "culture, instruction civique et loisirs" etc.)

**Comment by LVAI** [comment]
Cette allocation est différente de l'AAH (allocation adulte handicapé) je suppose ?
Est-ce qu'on ne peut pas la supprimer (Mélenchon veut augmenter l'AAH je crois, ça pourrait compenser un peu) en sachant qu'il n'aurait plus d'interprète à payer ?

**Comment by NoelleM** [comment]
Oui, cette subvention est ajoutée à l'AAH... Tu peux te faire une bonne idée de la réalité des personnes handicapées via ces sites  https://www.agefiph.fr/ et http://www.fiphfp.fr/

**Comment by LVAI** [comment]
@NoelleM pardon j'avais pas vu ta question dans le premier commentaire.
Non je pense qu'on peut faire les deux en parallèle. Comme ça on garde une trace.

**Comment by LucileDT** [comment]
Vous pourriez aussi proposer aux Sourds que vous connaissez de venir s'exprimer ici directement en leur donnant le lien, par exemple via des publications Facebook.

**Comment by LVAI** [comment]
@LucileDT oui pourquoi pas, ça peut se faire en effet ! C'est un peu l'objectif du formulaire (qui lui n'est pas estampillé JLM :wink: ) mais je crois que pour les Sourds s'exprimer en français reste souvent difficile. L'idéal pour eux c'est quand même la LSF.

**Comment by LucileDT** [comment]
Oui, malheureusement ! Ils pourraient se filmer et nous envoyer les vidéos, qu'on traduirait à l'écrit pour le rapport... Il nous manque réellement des contacts, c'est dommage :(

**Comment by LVAI** [comment]
Oui c'est ce que je propose dans ma vidéo (en espérant que je sois comprise...)
Ils nous en manque aujourd'hui mais on va travailler pour améliorer ça !

**Comment by LVAI** [comment]
Bonjour à tous, 
Je vous propose un petit résumé des résultats de notre sondage à ce jour.

Nombre de réponses : 46 
Répartition homogène hommes / femmes (46% contre 54%).
Des personnes de tous âges de mineur a sexagénaire (avec une plus forte représentation des 18-25 et des 35-45)
Une majorité de Sourds (65%)
Une bonne répartition sur la Métropole
![carte2](https://cloud.githubusercontent.com/assets/25056691/22243927/be6d2542-e229-11e6-9ccf-5aca31b5331a.png)

----------------------------
-- **Thèmes abordés** --
----------------------------

-- **Accessibilité à l'information**
**C'est un des thèmes phares**. Il revient dans plus d'une proposition sur 3.

Les demandes sont l'accessibilité en LSF de manière générale, et en particulier dans les médias : télévision (les émissions, le journal, les débats politiques mais plus largement toutes les chaines de la TNT 24/24) - les Sourds payent bien leur redevance TV ! -, internet, les diffusions en direct ou non, des encarts suffisamment grands.
Ils évoquent également les sous-titres (dont il faut améliorer la qualité), la vélotypie et le relais téléphonique (gratuit).

-- **Santé**

La santé est abordée plusieurs fois. Ils souhaitent pouvoir communiquer avec le personnel médical. Ils évoquent des unités d'accueil en LSF, des interprètes et médiateurs permanents dans les centres de soin.

J'en profite pour ajouter qu'aucune référence n'est faite à l'orthophonie, aux appareils et différentes opérations.

-- **Education**
Environs une proposition sur 5 en parle.

Les messages portent sur l'ouverture de nouvelles écoles bilingues, l'amélioration de la qualité de l'enseignement de la LSF (professeurs compétants), l’apprentissage en LSF pour toute la scolarité (maternelle à terminale puis université), l'adaptation de l'apprentissage du français aux besoins des enfants Sourds. Ils évoquent également l'ouverture de l'apprentissage _gratuit_ de la LSF : obligatoire pour tous ou pour ceux qui le souhaitent. Ainsi que des interventions pour sensibiliser les enfants à tous les handicaps et favoriser leur intégration.
Ils parlent également de l'éducation par le jeu (pour les Sourds ? Pour l'initiation à la LSF ?) et de mettre au programme de l'éducation nationale l'histoire des Sourds de France

-- **Emploi**
**Deuxième gros sujet** (plus d'une proposition sur 3).

Ils demandent l'embauche de Sourds signants dans des lieux publiques, la sensibilisation des entreprises et centres de formations, la formation au handicap dans les écoles de RH, de l'aide à l'insertion professionnelle, des formations gratuites pour les entreprises qui embauchent un sourd, 

Plusieurs personnes indiquent le métier auquel elles voudraient avoir accès : les arts graphiques, assistance sociale, maintenance informatique, aide soignant, photographie, travail auprès des parents d'enfants sourds.
Alors j'ajoute aussi, que les Sourds sont tout à fait capable de faire n'importe quel métier. Ils rencontrent en général des difficultés dans la formation (qu'ils s’efforcent de surmonter) et surtout à l'embauche.

-- **Economie**

Le prix de l'accessibilité revient plusieurs fois.
Ils demandent que la formation à la LSF soient gratuite pour les enfants sourds (devenus ou nés), moins chère voire gratuite pour les adultes et j'ajoute le problème des familles qui payent très cher leur formations ou apprennent sur le tas.

-- **Social**

La mise en place de lieux d'échanges et de regroupement est évoquée (café signe par exemple). Et il est rappelé combien il anormal de ne pas pouvoir communiquer avec quelqu'un juste parce qu'il est sourd.

-- **Services Publics**

Le besoin d'accessibilité dans les services et espaces publics est également rappelés à plusieurs reprises : permanences d'agents signants dans les services publics (Mairies, CCAS, Conseils Régionaux, Conseils du Département, MDPH, CAF, ...), présence de médiateurs culturels (de qualité) pour les visites guidés, signalisations écrites dans les gares, les métros, les  aéroports, 

-- **Enfance**

Mise en place de dispositifs dans les crèches pour l'accueil des petits Sourds.


Au-delà de ces sujets, le thème de l'égalité et du non handicap revient plusieurs fois. Les Sourds veulent avoir accès à l'information et pouvoir communiquer de manière simple dans leur vie de tous les jours. Ils insistent sur le fait que leur langue est différente, qu'il faut la reconnaître, mais qu'en dehors de cela ils sont normaux et ne souffrent d'aucun handicap.

Et je conclus sur le fait que certaines propositions sont difficilement exploitables. Elles ne répondent pas aux attentes du sondage ou sont peu claires. D'où l'importance de nous exprimer en LSF et de leur donner la possibilité de le faire.

Enfin,
35 personnes veulent recevoir les résultats du questionnaire
10 ont des idées pour améliorer l'accessibilité des vidéos YouTube
12 veulent participer au sous-titrage de vidéos YouTube
12 sont volontaires pour traduire des vidéos YouTube en LSF

**Comment by LVAI** [comment]
**Question :** Ce projet a-t-il un sens ?
Est-il utile que nous écrivions un rapport complet sur les axes d'amélioration que le gouvernement pourrait mettre en place concernant la communauté Sourde ?

J'ai bien conscience que ces questions ont déjà été mille fois posées au sein des groupes d'échanges et associations de Sourds. En la posant une 1001ème fois, ne sommes-nous pas en train de réinventer la roue ?
La question est donc : quel est notre objectif ? Est-ce nécessaire ?

Dans mon idée, notre démarche était de sensibilisernos futurs dirigeants à cette cause, ou à défaut de les interpeller, par l'intermédiaire des équipes de campagne des candidats - et en particulier JLM - dans l'espoir qu'il la prenne en considération dans leur campagne.
Je continue de penser que c'est important de le faire. Mais que pouvons nous vraiment faire ? Sommes-nous les mieux placer pour le faire ? Quelle est notre valeur ajoutée ? Comment pouvons nous être réellement utile ?

Faut-il écrire tout un rapport ? ou un simple e-mail ? Faut il mettre dedans les résultats du sondage et notre maigre expérience ? Ou simplement des renvoie vers les associations et groupes sachants (FNSF, etc...) ?

**Comment by LucileDT** [comment]
Peut-être qu'un livret thématique sur les Sourds devrait voir le jour. Et dans ce cas, il faut que j'en parler de vivre voix avec les gens qui rédigent le programme, donc le week-end du 5 février idéalement.

Alors, malheureusement, comme les Sourds sont marginalisés, il est possible que notre rôle de passerelle soit primordiale auprès des entendants non sensibilisés au sujet. Je préférerais que ce soient les Sourds qui prennent en main leur vie politique, oui, mais pour l'instant nous n'en avons qu'un seul qui soit affilié à la France Insoumise, c'est peu. Idéalement, si on pouvait permettre à un groupe de Sourds Insoumis de voir le jour, puis nous effacer pour leur laisser la place et leur permettre d'être les acteurs de leur mouvement, ça serait génial !

**Comment by LVAI** [comment]
@LucileDT j'approuve à 100% ton message.
Tiens-nous au courant des réponses que tu auras le week-end prochain et en particulier sur cette histoire de livret dédié aux Sourds.
